example-config.yml

```diff
@@ -362,6 +362,8 @@ itinerary:
   showScheduleDeviation: true
   # Shows the duration of a leg below the leg in the metro itinerary summary
   showLegDurations: false
+  # Whether to add a OTP_RR_A11Y_ROUTING_ENABLED error to all itineraries with accessibility scores
```

Comment:
Can we add a note here that `OTP_RR_A11Y_ROUTING_ENABLED` will also need to be added in config?

Reply:
I think it's clear if it's missing (big error in the html()